lib/secure_headers/configuration.rb

```diff
@@ -120,19 +120,36 @@ def deep_copy_if_hash(value)
 
     attr_reader :cached_headers, :csp, :cookies, :csp_report_only, :hpkp, :hpkp_report_host
 
+    @script_hashes = nil
+    @style_hashes = nil
+
     HASH_CONFIG_FILE = ENV["secure_headers_generated_hashes_file"] || "config/secure_headers_generated_hashes.yml"
-    if File.exists?(HASH_CONFIG_FILE)
+    if File.exist?(HASH_CONFIG_FILE)
       config = YAML.safe_load(File.open(HASH_CONFIG_FILE))
       @script_hashes = config["scripts"]
       @style_hashes = config["styles"]
     end
 
     def initialize(&block)
+      @cookies = nil
```

Comment:
This is kind of a bummer. Same for https://github.com/twitter/secureheaders/pull/330/files#diff-16a93318ee7771ce5483636b9959f4ec. Should we just not use `@instance_var_name`?